doc/builders/images/binarycache.section.md

```diff
@@ -0,0 +1,13 @@
+# pkgs.mkBinaryCache {#sec-pkgs-binary-cache}
+
+`pkgs.mkBinaryCache` is a function for creating Nix flat-file binary caches. Such a cache exists as a directory on disk, and can be used as a Nix substituter by passing `--substituter file:///path/to/cache` to Nix commands.
```

Comment:
Please make a full usage example, starting with such a derivation and using it on a full command line. While what you've written is in principle enough for an experienced user to abstract over, it is by far not detailed enough for a beginner to make sense of. Where does the binary cache end up? A full example will show the store path on stdout. How exactly do you use it as substituter? A full example will present a complete command line.

Also please link to the Nix manual for the relevant commands and flags if you mention them in the prose part, this makes discovery much easier.

Reply:
Okay, I've made the docs more descriptive. Could you take another look @fricklerhandwerk ?